docker/production-arm/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+# Use admin/pass as user/password credentials to login to openemr (from OE_USER and OE_PASS below)
+# MYSQL_HOST and MYSQL_ROOT_PASS are required for openemr
+# MYSQL_USER, MYSQL_PASS, OE_USER, MYSQL_PASS are optional for openemr and
+#   if not provided, then default to openemr, openemr, admin, and pass respectively.
+version: '3.1'
+services:
+  mysql:
+    restart: always
+    image: jsurf/rpi-mariadb
+    command: ['mysqld','--character-set-server=utf8mb4']
+    volumes:
+    - databasevolume:/var/lib/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+  openemr:
+    restart: always
+    image: openemr/openemr
```

Comment:
This won't work without somebody rebuilding :latest. Should we go ahead with that?

Reply:
That's a very good point. The script at production won't work either since 5.0.1 won't work with utf8mb4. And we can't rebuild 5.0.1 at this point since there's stuff in rel-502 that has not yet been released in a patch. I'm gonna put some wording in the documentation that these will not be production quality until 6.0.0 is released (and will place 6.0.0 tags on them in the image).